webui/react/src/pages/FlatRuns/FlatRuns.tsx

```diff
@@ -948,6 +948,35 @@ const FlatRuns: React.FC<Props> = ({ projectId, searchId }) => {
     };
   }, [canceler, stopPolling]);
 
+  const selectionLabel = useMemo(() => {
+    const newSettings: SelectionState = { ...settings.selection };
+    let runsList: number[] = [];
+
+    if (newSettings.type === 'ALL_EXCEPT') {
+      runsList = [...newSettings.exclusions];
+    } else {
+      runsList = [...newSettings.selections];
+    }
+
+    const numberOfSelection = runsList.length;
```

Comment:
actually we dont need to assign the array because we just wanna know the length.